django/contrib/humanize/locale/en/LC_MESSAGES/django.po

```diff
@@ -329,47 +329,47 @@ msgstr[0] ""
 msgstr[1] ""
 
 #. Translators: 'naturaltime-future' strings will be included in '%(delta)s from now'
-#: contrib/humanize/templatetags/humanize.py:214
+#: contrib/humanize/templatetags/humanize.py:262
 #, python-format
 msgctxt "naturaltime-future"
 msgid "%(num)d year"
 msgid_plural "%(num)d years"
 msgstr[0] ""
 msgstr[1] ""
 
-#: contrib/humanize/templatetags/humanize.py:215
+#: contrib/humanize/templatetags/humanize.py:265
 #, python-format
 msgctxt "naturaltime-future"
 msgid "%(num)d month"
 msgid_plural "%(num)d months"
 msgstr[0] ""
 msgstr[1] ""
 
-#: contrib/humanize/templatetags/humanize.py:216
+#: contrib/humanize/templatetags/humanize.py:268
 #, python-format
 msgctxt "naturaltime-future"
 msgid "%(num)d week"
 msgid_plural "%(num)d weeks"
 msgstr[0] ""
 msgstr[1] ""
 
-#: contrib/humanize/templatetags/humanize.py:217
+#: contrib/humanize/templatetags/humanize.py:271
 #, python-format
 msgctxt "naturaltime-future"
 msgid "%(num)d day"
 msgid_plural "%(num)d days"
 msgstr[0] ""
 msgstr[1] ""
 
-#: contrib/humanize/templatetags/humanize.py:218
+#: contrib/humanize/templatetags/humanize.py:274
 #, python-format
 msgctxt "naturaltime-future"
 msgid "%(num)d hour"
 msgid_plural "%(num)d hours"
 msgstr[0] ""
 msgstr[1] ""
 
-#: contrib/humanize/templatetags/humanize.py:219
+#: contrib/humanize/templatetags/humanize.py:277
```

Comment:
I don't see any new strings in `humanize`, please revert.

Reply:
I figured it was important for translators to have the updated comment. Is that the case?

Reply:
I don't think it's important, but it's your decision.